internal/closure/jstest/jstest.go

```diff
@@ -127,6 +127,6 @@ func TestCwd(t *testing.T) {
 	c := exec.Command("mocha", "-r", bootstrap, "-r", depsFile, filepath.Join(path, "*test.js"))
 	b, err := c.CombinedOutput()
 	if err != nil {
-		t.Fatalf(string(b))
+		t.Fatalf("mocha failed: %v\n%s", err, string(b))
```

Comment:
it doesn't matter in this case, but FYI with %s, both string and `[]byte` are handled the same way, so this `string(b)` is now redundant (and actually just allocates)

But we don't care about performance here.